### ISSUE TITLE
SC2

### ISSUE THREAD

**Issue by foehammer-419** [foehammer-419/Classes#7]
# Neuro Platinum SC2

VOD: https://www.reddit.com/r/allthingszerg/comments/94b1f8/neuro_coaching_platinum_zerg_macro_fundamentals/

General stuff

- 4:30 -5 minute spores is good depending on what you see

- 60 drones is a good time to make a round of ling/bane

- Creep around bases

- After the 3rd, when you drop both gasses, also add on another base

- Supply management

- Watch the injects

- Macro hatch at 6, 4th base at 8

ZvT

Mech

- Terran usually will deny the 4th easier than in ZvP

- You can usually make a 6 minute macro hatch, and get a 4th around 8:30

- 3:30 bane nest, 4:30 lair

- 29 OL, 40 OL

- Watch the supply blocks

- Carapace, ling, queen, is a good versatile opening

- 4 - 4:30 is a good time to sac the OL and make 2 or 3 OLs behind it.  

- Droning is often a bigger focus than the correct units

- Have up to 9 queens, 3 on hatches injecting, the rest on hotkey

- Mutas are good vs Bio but bad Vs mech

- Every time a new base finishes inject it with a queen

Bio

- If you do something successful, have something behind it/know what you're going to transition to.

- Have a plan of what you want to make

- Hydras you want to have a 50 drone econ, ling/bane can be 14-30 drone econ

- Scout consistently and react

ZvP

- In ZvP you want the 3rd around 30 supply

- With a 2 base all in, Identify what they're going and react (ling, bane, hydra if stargate) (Roach,Hydra, bane if archon/zealot)

- 4-4:30 lair and roach warren with evo chamber, hydra den when lair is done.  

- Queen hydra brood is the best max out composition

- basically hydra, bane, queen to start out with (harassing w/ small units of banes),

- Hive/spire should start around the same time

Plat 3 Coaching:

VOD: https://www.youtube.com/watch?v=ydTTPFwT_uo

Plat 1 Coaching:

https://www.youtube.com/watch?v=by-0FwS3mpc&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO&index=11

Diamond 1 Coaching:

VOD: https://www.youtube.com/watch?v=LlQ2Zmc9uIc

VOD: https://www.youtube.com/watch?v=VSCQ_3TLNfI&index=5&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO

VOD: https://www.youtube.com/watch?v=vQifsfjODLE&index=9&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO

**Comment by foehammer-419** [comment]
# Neuro Platinum SC2

VOD: https://www.reddit.com/r/allthingszerg/comments/94b1f8/neuro_coaching_platinum_zerg_macro_fundamentals/


General stuff

- 4:30 -5 minute spores is good depending on what you see

- 60 drones is a good time to make a round of ling/bane

- Creep around bases

- After the 3rd, when you drop both gasses, also add on another base

- Supply management

- Watch the injects

- Macro hatch at 6, 4th base at 8




ZvT

Mech

- Terran usually will deny the 4th easier than in ZvP

- You can usually make a 6 minute macro hatch, and get a 4th around 8:30

- 3:30 bane nest, 4:30 lair

- 29 OL, 40 OL

- Watch the supply blocks

- Carapace, ling, queen, is a good versatile opening

- 4 - 4:30 is a good time to sac the OL and make 2 or 3 OLs behind it.  

- Droning is often a bigger focus than the correct units

- Have up to 9 queens, 3 on hatches injecting, the rest on hotkey

- Mutas are good vs Bio but bad Vs mech

- Every time a new base finishes inject it with a queen


Bio

- If you do something successful, have something behind it/know what you're going to transition to.

- Have a plan of what you want to make

- Hydras you want to have a 50 drone econ, ling/bane can be 14-30 drone econ

- Scout consistently and react




ZvP

- In ZvP you want the 3rd around 30 supply

- With a 2 base all in, Identify what they're going and react (ling, bane, hydra if stargate) (Roach,Hydra, bane if archon/zealot)

- 4-4:30 lair and roach warren with evo chamber, hydra den when lair is done.  

- Queen hydra brood is the best max out composition

- basically hydra, bane, queen to start out with (harassing w/ small units of banes),

- Hive/spire should start around the same time



Plat 3 Coaching:

VOD: https://www.youtube.com/watch?v=ydTTPFwT_uo

General Notes:


- Every time you attack take a base

- Either wound protoss, or hive tech fast, generally defend w/ hydra

- w/ each new base, single spore double spine

- Shoot for 70 workers ideally, but ~60 is fine




ZvP

- Scout

- Use the units that you have

- 16 workers in minear line(s)

- Every time you attack take a base

- Main ZVP Options:

A.  Wound protoss and macro

- ling/bane attack/harass, and get infestation pit, defend w/ hydras

B.  Rush Hive tech and kill them

- Defend w/ everything you can


- Generally since toss units are better zerg should be able to hit 200 faster.  

- At ~170 zerg should hit toss and make units behind the attack that are higher tech (ultras or brood lords)


- Broodlords are generally better against toss, unless they are going into stargate play


ZvP (Cannon Rush)

- Analyze how much they're investing in the rush, is it worth it to build another hatch far away and nydus rush them?

- put the pool behind the mineral line (This also works for ZvT to prevent hellions)

- 2nd hatch should be far away/randomly placed on the map (usually at the extremes is best)

- Ling speed with 1st 100 gas, then lair after 1st queen

- Shoot for 7 queens

ZvT

- 1st OL goes to opp nat, 2nd OL goes to your 3rd and then to opp main

- W/ Marine medivac marine you need queens/lings to defend (7-8 queens, 30+ lings), drone until ~4:30

- With Terran, hydras are a small number support unit for ling/bane, against Protoss you can mass hydras a bit more to get to BL since hydras counter more of their untis


- 1 range and carapace, and then 1 melee and 2 carapace


- Terran is about, do you have enough units of the right kind at the right place to defend?

- A good # of Hydras for ZvT is 14.  

- You want to phase out of ling/bane over time and into hive tech units, because ling/bane starts to decrease in value when Terran gets a critical mass of tanks or liberators (which are common Terran transitions)

- Ultra corruptor is better than ultra hydra because of pathing

- When getting to ultra tech, make a round of queens, and pull your existing queens to go with the army and transfuse your ultras

- usually it is more powerful to tech/expand than it is to make more army/feel safe



ZvT Mech:

- There is more value in double evo, b/c 2/2 is around when your army is maxed out and therefore more units have the upgrades

- Start Infestation pit as soon as you start making hydras, and then make hive as soon as the infestation pit finishes

- For hellion defense, active/attacking queens should be at the latest expansion (generally 3rd), and all other units should be at the natural.  

- Go for 70 drones

- Try not to make hydras until ~70 drones

- For composition against mech, a good one to shoot for is ultra corruptor off of 5 base/70 drones.  Roach hydra as strong midgame options

Plat 1 Coaching:

https://www.youtube.com/watch?v=by-0FwS3mpc&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO&index=11




Diamond 1 Coaching:

VOD: https://www.youtube.com/watch?v=LlQ2Zmc9uIc

VOD: https://www.youtube.com/watch?v=VSCQ_3TLNfI&index=5&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO

VOD: https://www.youtube.com/watch?v=vQifsfjODLE&index=9&list=PLubOCgpR4-77cLw6gX68zf3z27jAjE4wO
